front/lib/strapi/strapiCRUD.ts

```diff
@@ -3,8 +3,25 @@ import qs from 'qs';
 
 const base_url = 'http://localhost:1337/api/';
```

Comment:
you can create file /strapi/constants.ts
and in this file write export const BASE_URL=...

this will allow you to store const in easier to extract way.

The best option in my mind would be creating .env file in root frontend dir, and there writing NEXT_PUBLIC_BASE_URL"... Then you can access variable via process.env.NEXT_PUBLIC_BASE_URL
https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables